internal/compiler-interface/src/main/java/xsbti/ArtifactInfo.java

```diff
@@ -15,11 +15,11 @@
 	public static final String ScalaOrganization = "org.scala-lang";
 
 	/** Define the name used to identify the sbt organization. */
-	public static final String SbtOrganization = "org.scala-sbt";
+	public static final String SbtOrganization = "ch.epfl.scala";
```

Comment:
Why is this required? :smile:

Reply:
Otherwise the tests won't find the bridge's sources 😢

Reply:
This will need to be removed when we try to merge upstream, then :wink: